doc/sphinx/language/gallina-extensions.rst

```diff
@@ -695,48 +695,44 @@ terminating functions.
 
 .. seealso:: :ref:`functional-scheme` and :tacn:`function induction`
 
-Depending on the :n:`@fixannot` annotation, different definition mechanisms are
-used by ``Function``. A more precise description is given below.
+.. todo I think everything from here to "Section mechanism" should appear before the Note above
```

Comment:
Sounds good!

Reply:
Moved it.  Our style seems to be to indent the description of commands and tactics, but it gets to be a long indentation, probably more than a page, which kinda defeats the purpose.  Perhaps worth reconsidering at some point whether such indentation is best.